### PR TITLE
Mark apple watch podcasts app as bot

### PR DESCRIPTION
**Pull request by cavis**
Merge after #25.

https://iabtechlab.com/blog/apple-watch-os-podcast-filtering-guidance/

### CODE REVIEW

File: db/agents.json [hunk 1]
```diff
@@ -1610,13 +1712,18 @@
     },
     {
       "regex": "^(AppleCoreMedia.+Apple Watch|atc.+watchOS|\\(null\\).+watchOS)",
```
Comment (kookster) [inline db/agents.json:1714]: this is more expansive than the IAB guidelines
https://iabtechlab.com/blog/apple-watch-os-podcast-filtering-guidance/
```
User Agents:
That begin with atc/ and include watchOS (for example atc/1.0 watchOS)
Or contain (null)/(null) watchOS*
```
Looks like you added filtering out `AppleCoreMedia.+Apple Watch` too

**Review by kookster — CHANGES_REQUESTED** [review]
I think our regex for apple watch is too expansive

File: db/agents.json (hunk 1 above)
Comment (kookster) [reply db/agents.json:1714]: I mean, I agree those are Apple Watch, for identifying
Not sure they should be filtered out - that might be the "legit" traffic from them.

**Comment by kookster** [comment]
ha! we both merged master at the same time!

File: db/agents.json (hunk 1 above)
Comment (cavis) [reply db/agents.json:1714]: Good point - will separate those out.

Though FWIW - they're actually very little of our Apple watch traffic.  Counts for November:

```
114602332 -> all downloads
29272     -> remote_agent like 'AppleCoreMedia%Apple Watch%'
7100376   -> remote_agent like 'atc%watchOS%'
16214     -> remote_agent like '(null)%watchOS%'
```

Comment (kookster) [reply db/agents.json]: cool cool cool

File: db/agents.json [hunk 2]
```diff
@@ -1711,14 +1711,27 @@
...
+      "regex": "^AppleCoreMedia.+Apple Watch",
+      "name": "Apple Podcasts",
+      "type": "Smart Watch",
+      "os": "WatchOS",
+      "examples": [
+        "AppleCoreMedia/1.0.0.16S535 (Apple Watch; U; CPU OS 5_1_3 like Mac OS X; en_us)"
+      ]
+    },
+    {
+      "regex": "^(atc.+watchOS|\\(null\\).+watchOS)",
```
Comment (kookster) [inline db/agents.json:1723]: nice

**Review by kookster — APPROVED** [review]
niiiiice

**Pull request merged**